shedding_hub/shedding_duration.py

```diff
@@ -384,6 +385,7 @@ def plot_shedding_durations(
                 row["shedding_duration_mean"], study_counter, marker="o", color=color
             )
             study_counter += 1
+        assert line, f"Group for specimen '{name}' is empty."
```

Comment:
This is not technically necessary, but it just makes sure that `line` is always defined and has been populated in the loop. Otherwise, some of the automated type checking in VS Code may complain.